### ISSUE TITLE
DelfinovinUI Window not showing, process exist

### ISSUE THREAD

**Issue by Rinkaaz** [Struggleton/Delfinovin#59]
Hello, 

I reinstalled my drivers, .NET 7.0.11 and after the windows alert and exectution, the software not appears. 
I'm on Windows11, do you guys have any idea ? 

Thanks you

**Comment by IoIxD** [comment]
It doesn't launch when you try to start it from the start menu, for me. You'll have to start it from the downloads folder.

**Comment by ManyMilesAway** [comment]
Having this same issue, Windows 11. It doesn't know when it wants to open or not.

**Comment by SkullLingus** [comment]
Same here, it was working fine till now (maybe a windows update). I double click the exe and nothing happen but it does seem to appear in the processes

**Comment by MattTreichelYeah** [comment]
Just downloaded this and seeing it as well 🤔 The first time I ran it, it prompted me to download .NET and I did, but after doing that, no behaviour happens when running the .exe (after bypassing the security prompt). On Windows 11 as well.

edit: ok, duh, I guess I just need to "Run as Administrator". No prompt for that or anything though.

**Comment by ThomasH94** [comment]
After months of trying I was able to fix this!
I was also having this issue, and _I believe_ it's an issue with checking for updates - as none are available.

I updated the "settings.txt" and that worked for me! I hope it works for you too!

Here is my settings file:

MinimizeToTray: True
MinimizeOnStartup: True
CheckForUpdates: False
DefaultProfile1: 
DefaultProfile2: 
DefaultProfile3: 
DefaultProfile4: 
ApplicationTheme: 
ControllerColor: 


Best of luck everyone!

**Comment by SirPrimalform** [comment]
Hmm, I'm having this problem too. Strangely, on first trying ThomasH94's solution it started up reliably multiple times in a row. But after that it's reverted to the old behaviour of starting but never showing the GUI. Really strange, hope someone else has some insight.

**Comment by MattTreichelYeah** [comment]
Yeah, I thought I fixed it but it's just not consistent at all. I usually resort to either freshly downloading it every time or now running the v2 version and ignoring the prompt to update.

**Comment by TicTac7x** [comment]
You can get 3.0 working properly by downgrading the ViGEm driver: https://github.com/nefarius/ViGEmBus/releases/download/v1.18.367.0/ViGEmBus_1.18.367_x64_x86.exe

At least this worked for me, Delfinovin opens every time properly now.
